hazelcast/src/main/java/com/hazelcast/jet/impl/util/Util.java

```diff
@@ -137,7 +136,7 @@ public static boolean tryIncrement(AtomicInteger value, int increment, int limit
     }
 
     public static JetInstance getJetInstance(NodeEngine nodeEngine) {
-        return nodeEngine.<JetService>getService(JetService.SERVICE_NAME).getJetInstance();
+        return nodeEngine.<JetServiceBackend>getService(JetServiceBackend.SERVICE_NAME).getJetInstance();
```

Comment:
I guess we could make this method return the new `JetService` to stop using the deprecated APIs internally.